plugins/nodes/src/js/columns/NodesTableCPUColumn.tsx

```diff
@@ -7,8 +7,7 @@ import { IWidthArgs as WidthArgs } from "#PLUGINS/nodes/src/js/types/IWidthArgs"
 import { SortDirection } from "plugins/nodes/src/js/types/SortDirection";
 
 export function cpuRenderer(data: Node): React.ReactNode {
-  // TODO: DCOS-38822
-  return <span>{data.get("used_resources").cpus.toString()}</span>;
+  return `${data.getUsageStats("cpus").percentage}%`;
 }
 export function cpuSorter(data: Node[], sortDirection: SortDirection): Node[] {
   // TODO: DCOS-38822
```

Comment:
There is a comment with this ticket number missing and the sorting is inverse of what the old implementation provides

Reply:
hm, you're right, not the question is: was the old one intentional or was it a bug? 🤔

Reply:
No, unintentional

Reply:
It seems like github does not allow me to comment on the right side, but only here

Reply:
after slack discussion: we keep it as is 👍